examples/README.md

```diff
@@ -3291,24 +3341,57 @@ We fist define our top-level workflow for this example:
        },
        {
           "name": "DoCarVitalsChecks",
-          "type": "subflow",
+          "type": "operation",
+          "actions": [
+            {
+              "subFlow": {
+                "workflowId": "vitalscheck",
+                "waitForCompletion": false
```

Comment:
if we allow async exec for subflow actions it should be also possible for functionRef actions